.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> Sure, I need your advice on how best to do this then

It depends on whether or not we really want to make Android Studio a prerequisite of the tutorials. I think it would be much simpler for the tutorial simply to be a gradle commandline project. Then anyone looking at it can see easily what's there without all the boilerplate cruft, configuration, project files, etc